instrumentation/message-handler/aws/build.gradle.kts

```diff
@@ -0,0 +1,14 @@
+plugins {
```

Comment:
usually we add the minimum supported version of the library to the module name

Reply:
And also we place code in a directory named `library` (or `javaagent` depending on what the module does).